components/collapsed_reply_threads_modal/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export {default} from './collapsed_reply_threads_modal';
```

Comment:
We don't actually need this file if we're not connecting to redux or passing in any props or actions using it.